web/client/product/plugins.js

```diff
@@ -124,7 +125,13 @@ export default {
         WidgetsTrayPlugin: require('../plugins/WidgetsTray').default,
         ZoomAllPlugin: require('../plugins/ZoomAll').default,
         ZoomInPlugin: require('../plugins/ZoomIn').default,
-        ZoomOutPlugin: require('../plugins/ZoomOut').default
+        ZoomOutPlugin: require('../plugins/ZoomOut').default,
+        PrintTextInputPlugin: require('../plugins/print/TextInput').default,
+        PrintLayoutPlugin: require('../plugins/print/Layout').default,
+        PrintLegendOptionsPlugin: require('../plugins/print/LegendOptions').default,
+        PrintResolutionPlugin: require('../plugins/print/Resolution').default,
+        PrintMapPreviewPlugin: require('../plugins/print/MapPreview').default,
+        PrintOptionPlugin: require('../plugins/print/Option').default
```

Comment:
My idea was not to include any plugin, but only allow to override by plugin injection, on demand. Having this big set of plugin to configure or customize a single plugin is a problem. Potentially makes the size of the plugins list explode. 

I asked to have a plugin like : 
```javascript 
{
    "name": "PluginCustomization",
    "containers": {
            "Print": {
                    target: "left-panel-accordion", // I suggested to have specific targets to replace specific components.
                    position:  "" 
                    Component: MyCustomization,
            }
    }
}
```